Base/QTGUI/qSlicerApplication.h

```diff
@@ -101,6 +101,10 @@ class Q_SLICER_BASE_QTGUI_EXPORT qSlicerApplication : public qSlicerCoreApplicat
   Q_INVOKABLE qSlicerLayoutManager* layoutManager()const;
   Q_INVOKABLE void setLayoutManager(qSlicerLayoutManager* layoutManager);
 
+  /// Set/Get operator name
+  Q_INVOKABLE QString operatorName()const;
+  Q_INVOKABLE void setOperatorName(const QString &name);
```

Comment:
Thanks for your contribution.

Could you:

* provide a little more context by describing what is an `OperatorName` ?

* move the method into the `qSlicerCoreApplication`

* Remove the use of `Q_INVOKABLE` and instead use `Q_PROPERTY`